### DISCUSSION TITLE
Is there a quick start to get CrashPlan running on Container Manager instead of Docker?

### DISCUSSION THREAD

**Discussion by OingoFan55** [jlesage/docker-crashplan-pro#457] [Q&A]
I upgraded my Synology NAS and found that they want you to use Container Manager instead of using Docker.  I had CrashPlan running relatively smoothly on my old NAS through Docker, but now want to set it up to use Container Manager...and it seems like some have it running fine.  Was looking for a quick start (or how-to) get CrashPlan setup and running on Container Manager.  Any tips?  Thanks!

**Comment by jcormier** [comment]
Container Manager is using docker.  Its just a "GUI"  that configures docker behind the scenes.  This video may help.
https://www.youtube.com/watch?v=X0qGNgmCIGw (How to use Docker (Container Manager) to do ANYTHING! on Synology NAS)

**Reply by OingoFan55** [reply]
I didn't understand the difference.  In the end, you install CrashPlan in the same method...even using the same installation as prior.  Container Manager is used after that (instead of Docker).  From what I see, it has better gui interface than what Docker had.  However, I had to install portainer in order to change my volume directories that had spaces in them.

**Reply by jcormier** [reply]
docker does not have a GUI. perhaps Synology made a GUI and named it docker and thus are creating this confusion?  Regardless sounds like you found a solution.
